lilypond/songs/love-divine-all-loves-excelling/love-divine-all-loves-excelling.ly

```diff
@@ -0,0 +1,55 @@
+\version "2.22.1"
+#(ly:set-option 'relative-includes #t)
+\include "../../lib/hymn-common.ily"
+
+%% TUNE INFO
+\include "../../shared-tunes/hyfrydol-public.ily"
+quarternoteTempo = 120
+
+%% SONG INFO
+title = \titleText "Love Divine, All Loves Excelling"
+poet = \smallText "Text: Charles Wesley, 1747"
+typesetter = "Zachary Cline"
+verseCount = 4
+tags = "english christian 4part"
+dateAdded = "2022-05-12"
+\include "../../lib/header.ily"
+
+%% LYRICS
+verseA = \lyricmode {
+  \l Love di -- vine, all loves ex -- cel -- ling, Joy of heaven to earth come down,
```

Comment:
I prefer "heav'n" for the single syllable form.

Reply:
done!